sdk/core/azure-core/azure/core/pipeline/transport/_aiohttp.py

```diff
@@ -525,16 +541,9 @@ def text(self, encoding: Optional[str] = None) -> str:
             elif body is None:
                 raise RuntimeError("Cannot guess the encoding of a not yet read body")
             else:
-                try:
-                    import cchardet as chardet
-                except ImportError:  # pragma: no cover
-                    try:
-                        import chardet  # type: ignore
-                    except ImportError:  # pragma: no cover
-                        import charset_normalizer as chardet  # type: ignore[no-redef]
                 # While "detect" can return a dict of float, in this context this won't happen
                 # The cast is for pyright to be happy
-                encoding = cast(Optional[str], chardet.detect(body)["encoding"])
+                encoding = cast(Optional[str], detect(body)["encoding"])
```

Comment:
Assuming no value for `encoding` is being passed in, what value of content-type/mimetype is used for your responses?

Entering this `else` statement _should_ be a pretty rare occurrence - so worth confirming that there's a way we can avoid importing chardet at all.